pkg/crypto/certloader/watcher.go

```diff
@@ -90,7 +91,10 @@ func FutureWatcher(ctx context.Context, log *slog.Logger, caFiles []string, cert
 			res <- w
 			return
 		}
-		log.Debug("Waiting on fswatcher update to be ready")
+		log.Warn("Initial TLS configuration load failed, certificate files might not yet be available, waiting on fswatcher update to become ready",
```

Comment:
If this is a permanent fixture, should we just make this a constant and use it in the cilium-cli code so it doesn't get out of sync in the future?